index.d.ts

```diff
@@ -489,6 +489,9 @@ declare module "skygear" {
 }
 
 declare module "skygear/cloud" {
+  import { Fields, Files } from "formidable";
```

Comment:
It seems that the definition is small enough that we can copy instead.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/formidable/index.d.ts#L32

I would like to reduce the number of dependencies.